src/test/java/org/jenkinsci/plugins/maven_artifact_choicelistprovider/azure/AzureDevopsFeedApiSearchServiceTest.java

```diff
@@ -0,0 +1,25 @@
+package org.jenkinsci.plugins.maven_artifact_choicelistprovider.azure;
+
+import org.jenkinsci.plugins.maven_artifact_choicelistprovider.ValidAndInvalidClassifier;
+import org.jenkinsci.plugins.maven_artifact_choicelistprovider.VersionReaderException;
+import org.jenkinsci.plugins.maven_artifact_choicelistprovider.nexus3.Nexus3RestApiSearchService;
+import org.junit.After;
+import org.junit.Test;
+
+import java.util.List;
+
+
+public class AzureDevopsFeedApiSearchServiceTest {
+
+    @Test
+    public void testWithoutExplicitQualifier() throws VersionReaderException {
+        AzureSearchService s = new AzureSearchService("https://acme.pkgs.visualstudio.com/<a-GUID>",true);
+        s.setUserName("tbs");
+        s.setUserPassword("tbd");
+        List<String> retrieveVersions = s.retrieveVersions("anazurefeedname", "com.acme", "syrupapp", "",
+                ValidAndInvalidClassifier.getDefault());
+        for (String current : retrieveVersions) {
+            System.out.println(current);
```

Comment:
Unsure how to properly provide a working test for this as I can't find any public azure devops repos (I've tested against a private and then the request works).